app/views/core/SubscribeModal.coffee

```diff
@@ -124,111 +88,112 @@ module.exports = class SubscribeModal extends ModalView
       @$el.find('.popover-content .parent-input').parent().addClass('has-error')
       @$el.find('.popover-content .parent-email-validator').show()
       return false
-
-    request = @supermodel.addRequestResource 'send_one_time_email', {
-      url: '/db/user/-/send_one_time_email'
-      data: {email: email, type: 'subscribe modal parent'}
-      method: 'POST'
-    }, 0
-    request.load()
-
+    me.sendParentEmail(email)
+    
     @$el.find('.popover-content .email-parent-form').hide()
     @$el.find('.popover-content .email-parent-complete').show()
     false
 
   onClickParentEmailCompleteButton: (e) ->
-    @$el.find('.parent-button').popover('hide')
+    @$el.find('.parent-link').popover('hide')
 
   onClickPurchaseButton: (e) ->
     return unless @basicProduct
     @playSound 'menu-button-click'
     return @openModalView new CreateAccountModal() if me.get('anonymous')
     application.tracker?.trackEvent 'Started subscription purchase'
-    options = {
+    options = @stripeOptions {
       description: $.i18n.t('subscribe.stripe_description')
       amount: @basicProduct.get('amount')
-      alipay: if me.get('country') is 'china' or (me.get('preferredLanguage') or 'en-US')[...2] is 'zh' then true else 'auto'
-      alipayReusable: true
     }
-
+    
     @purchasedAmount = options.amount
-    stripeHandler.open(options)
-
-  onClickSaleButton: (e) ->
+    stripeHandler.openAsync(options)
+    .then ({token}) =>
+      @state = 'purchasing'
+      @render()
+      window.jqxhrs ?= []
```

Comment:
What's window.jqxhrs for?  If this is something we always do with our jquery xhrs, mabye we should wrap jquery?  Perhaps a discussion for another day.